### ISSUE TITLE
Images are added to the article in a very misshaped size

### ISSUE THREAD

**Issue by vongohren** [gunar/medium-converter#12]
Feks: https://media.consensys.net/blockchain-vs-distributed-ledger-technologies-1e0289a87b16

The pdf of this one becomse like the following pdf. With huge pictures which cuts of over half of the picture itself.
[media.consensys.net-blockchain-vs-distributed-ledger-technologies.pdf](https://github.com/gunar/medium-converter/files/2281127/media.consensys.net-blockchain-vs-distributed-ledger-technologies.pdf)

**Comment by Clovis-Portron** [comment]
I don't think it depends of this project but of Docverter, since this project does nothing more than retrieving Markdown from Medium and passing it to docverter, which manage the layout.

**Comment by vongohren** [comment]
@Clovis-Portron well i went here: https://clovis-portron.github.io/medium-converter/
And from there i got a PDF on the same medium article, with no images.
So I dont know what is best, to big or none :P

**Comment by Clovis-Portron** [comment]
So it's definitely an issue with medium-parser, so let's wait for @DanielArthurUK's magic. :D

**Comment by DanielArthurUK** [comment]
Yes this is an issue I highlighted with medium-parser, see gunar/medium-parser#13

I will be opening a PR to fix this in the coming days, but it is likely this fix will come in the form of a major version to fix other issues that currently exist with this library. The major version will change the way the library is used so you may have to change some of your existing code if you wish to upgrade.

Also, thank you for believing in me @Clovis-Portron 😂 I am not sure if it will be magic but hopefully it will work ✨

**Comment by DanielArthurUK** [comment]
@Clovis-Portron just an update on this issue - I opened a PR last week for medium-parser, just waiting for @gunar to review and hopefully merge 🤞 Once that's done I'll open a PR here to update medium-converter to the new version. There are a bunch of other features too, feel free to check out my PR: gunar/medium-parser#15

**Comment by Clovis-Portron** [comment]
Great job! I will take a look :D
